asb-ballerina/Module.md

```diff
@@ -1,473 +1,262 @@
-```
+            string stringContent = "This is My Message Body"; 
+            byte[] byteContent = stringContent.toBytes();
+            int timeToLive = 60; // In seconds
 
-### Step 8: Receive a Message from Azure Service Bus
-You can now receive a message from the configured azure service bus entity.
-```ballerina
-    asb:Message|asb:Error? messageReceived = asbClient->receive(queueReceiver, serverWaitTime);
-
-    if (messageReceived is asb:Message) {
-        log:printInfo("Reading Received Message : " + messageReceived.toString());
-    } else if (messageReceived is ()) {
-        log:printError("No message in the queue.");
-    } else {
-        log:printError("Receiving message via Asb receiver connection failed.");
-    }
-```
+            asb:ApplicationProperties applicationProperties = {
+                properties: {a: "propertyValue1", b: "propertyValue2"}
+            };
 
-### Step 9: Close Sender Connection
-You can now close the sender connection.
-```ballerina
-    checkpanic asbClient->closeSender(queueSender);
-```
+            asb:Message message = {
+                body: byteContent,
+                contentType: asb:TEXT,
+                timeToLive: timeToLive,
+                applicationProperties: applicationProperties
+            };
 
-### Step 10: Close Receiver Connection
-You can now close the receiver connection.
-```ballerina
-    checkpanic asbClient->closeReceiver(queueReceiver);
-```
+            check asbClient->send(queueSender, message);
+
+            check asbClient->closeSender(queueSender);
+        }
+    ```
+
+    Following is an example on how to receive messages from the Azure Service Bus using the client connector. You need to create a 
+    `queueReceiver` before receiving a message. Optionally you can provide the receive mode which is PEEKLOCK by default. You can find more information about the receive modes [here](https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.receivemode?view=azure-java-stable).
+
+    Receive a message from the Azure Service Bus
+
+    ```ballerina
+        public function main() returns error? {
+            handle queueReceiver = check asbClient->createQueueReceiver(queueName, asb:RECEIVEANDDELETE);
+
+            int serverWaitTime = 60; // In seconds
+
+            asb:Message|asb:Error? messageReceived = asbClient->receive(queueReceiver, serverWaitTime);
+
+            if (messageReceived is asb:Message) {
+                log:printInfo("Reading Received Message : " + messageReceived.toString());
+            } else if (messageReceived is ()) {
+                log:printError("No message in the queue.");
+            } else {
+                log:printError("Receiving message via Asb receiver connection failed.");
+            }
+
+            check asbClient->closeReceiver(queueReceiver);
+        }
+    ```
+
+    Following is an example on how to asynchronously listen to messages from the Azure Service Bus using the listener. 
```

Comment:
Shouldn't we add these as full  fledged samples and not in the module.md? Please follow the same format as other connectors.

Reply:
This is a very important scenario. So it's important to add it here. We have agreed to give two examples if there is a client and a listener. Nothing else are added.